src/environment/stella_environment.cpp

```diff
@@ -178,6 +178,12 @@ void StellaEnvironment::soft_reset(){
   m_state.incrementFrame();
 }
 
+/** Emulates a NOOP action for both players */
+void StellaEnvironment::wait()
```

Comment:
@pvchaumier , I am just trying to avoid adding a new feature in the Stella environment, because this becomes a bigger change than just adding a game. @mgbellemare, can you weigh on that?